community/assessments/projects/k8gb/self-assessment.md

```diff
@@ -0,0 +1,234 @@
+    - Creates DNSEndpoint to configure DNS zone delegation in an external DNS provider.
+
+### Goals
+
+The intended goals of k8gb, including the security guarantees it provides.
+
+#### Secure and Verified Builds
+
+Ensure all K8GB releases are signed and verified to guarantee authenticity and integrity, protecting users from tampered or malicious builds.
+
+#### Minimal Attack Surface
+
+Expose only the necessary ports for GSLB operations, such as DNS (53/tcp and 53/udp), to reduce the attack surface and enhance security.
+
+#### Secure Deployment Practices
+
+Provide secure default configurations and documentation to help users deploy K8GB in a way that aligns with Kubernetes security best practices.
+
+These goals aim to make K8GB a reliable and secure solution for global load balancing while minimizing risks and ensuring trust in the project's artifacts.
+
+See also [Intended use](#intended-use).
+
+### Non-goals
+
+Non-goals that a reasonable reader of k8gb’s documentation could believe may be in scope.
+The k8gb project is focused on providing global load balancing and failover capabilities for Kubernetes applications. However, there are certain security-related features and responsibilities that are intentionally out of scope. Below are the non-goals, along with explanations for why they are not within the project's scope:
+
+#### Traffic Handling and TLS Termination:
+
+k8gb does not pass application traffic through itself or handle TLS/HTTP connections directly. These responsibilities are offloaded to referenced networking resources, such as Kubernetes Ingress controllers or service meshes.
+
+**Reason**: k8gb is designed to manage DNS-based global load balancing and failover, not to act as a proxy or gateway for application traffic.
+
+#### Security of Referenced Networking Resources:
+
+k8gb does not enforce or manage security configurations (e.g., TLS settings) for the networking resources it references, such as Ingress controllers.
+
+**Reason**: Security configurations for these resources are the responsibility of the user or the specific networking solution being used.
+
+## Self-assessment use
+
+This self-assessment is created by the k8gb team to perform an internal analysis of the project's security. It is not intended to provide a security audit of k8gb, or function as an independent assessment or attestation of k8gb's security health.
+
+This document serves to provide k8gb users with an initial understanding of k8gb's security, where to find existing security documentation, k8gb plans for security, and general overview of k8gb security practices, both for development of k8gb as well as security of k8gb.
+
+This document provides the CNCF TAG-Security with an initial understanding of k8gb to assist in a joint-assessment, necessary for projects under incubation. Taken together, this document and the joint-assessment serve as a cornerstone for if and when k8gb seeks graduation and is preparing for a security audit.
```

Comment:
All the TAG Security mentions should use the new name TAG Security and Compliance